pandas/tests/series/test_apply.py

```diff
@@ -17,6 +18,33 @@
 from .common import TestData
 
 
+def _get_cython_table_params(series, func_names_and_expected):
```

Comment:
I guess I would like this to be a centralized function, can you move to pandas.util.testing (as a private function like you have now), and consolidate with the one from frame).

Reply:
ok, done.